sample/todo-sync/src/com/cloudant/todo/TasksModel.java

```diff
@@ -51,7 +52,7 @@
     private static final String DATASTORE_MANGER_DIR = "data";
```

Comment:
Thanks for the feedback, I've fixed all the other incorrect use of datastore in comments etc.